File: raspberry-pi-ios-client/RaspberryPiClient/Data/Repository/Connection/ConnectionMode.swift
//
//  ConnectionMode.swift
//  RaspberryPiClient
//
//  Created by Shusuke Ota on 2025/03/21.
//

import Foundation

/// 接続先の Bridge 種別
enum ConnectionMode: String, CaseIterable {
    case rosBridge = "ROS Bridge"
    case foxgloveBridge = "Foxglove Bridge"
}

File: raspberry-pi-android-client/app/src/main/java/com/shusuke/raspberry_pi_android_client/data/repository/connection/ConnectionMode.kt
package com.shusuke.raspberry_pi_android_client.data.repository.connection

/**
 * 接続先の Bridge 種別
 */
enum class ConnectionMode(val displayName: String) {
    ROS_BRIDGE(displayName = "ROS Bridge"),
    FOXGLOVE_BRIDGE(displayName = "Foxglove Bridge");
}
